std_msgs/Int16 target_led_num # target led from 1 to 8 (1 is the end effector's)
---
std_msgs/Int16 current_intensity # from 0 to 255, -1 if state wasn't set before, or -2 if Id doesn't exist
